test/clj/chronograph/domain/timer_test.clj

```diff
@@ -0,0 +1,480 @@
+(ns chronograph.domain.timer-test
+  (:require [chronograph.domain.timer :as timer]
+            [clojure.test :refer :all]
+            [chronograph.fixtures :as fixtures]
+            [chronograph.factories :as factories]
+            [chronograph.domain.acl :as acl]
+            [chronograph.db.core :as db]
+            [clojure.spec.alpha :as s]
+            [chronograph.db.time-span :as db-time-span])
+  (:import org.postgresql.util.PSQLException))
+
+(def test-context
+  (atom {}))
+
+(defn- setup-members-org-tasks!
```

Comment:
Call this as a function in your test so that:
1. You avoid being unable to ||ize your tests because of the atom.
2. The test setup is in the test instead of outside it.

Reply:
Done.